src/FlatBufferCreator.h

```diff
@@ -13,7 +13,7 @@ struct EpicsPVUpdate;
 class FlatBufferCreator {
 public:
   virtual ~FlatBufferCreator() = default;
-  virtual std::unique_ptr<FlatbufferMessage>
+  virtual std::unique_ptr<FlatBufs::FlatbufferMessage>
```

Comment:
`FlatBufferCreator` is in the `FlatBufs` namespace, so don't need `FlatBufs::` on `FlatbufferMessage`